writehat/components/TitlePageShort.py

```diff
@@ -7,6 +7,8 @@
 
 
 class Component(TitlePageComponent):
+    class CustomDateWidget(forms.SelectDateWidget):
```

Comment:
does this need to be declared again since TitlePageComponent already declares this custom widget?

Reply:
Yes, I wanted to scope it to each Component class so as to keep the two components decoupled from each other.